From 14a888e01c6b25749b1ae456eec488671832e411 Mon Sep 17 00:00:00 2001
From: farenl <farenl@codeaurora.org>
Date: Tue, 24 Apr 2018 14:37:27 +0800
Subject: [PATCH 1/3] Calendar: Some fonts display too small in Calendar

Change Theme style from "Holo" to "Material" since "Holo" is
marked @deprecated Use Material themes on API 21+ or AppCompat
on supported APIs, which is introduced on P.

Change-Id: Ic46de3b92e8e5ad913818600a15d841e740c5305
CRs-Fixed: 2227536
---
 AndroidManifest.xml   | 10 +++++-----
 res/values/styles.xml |  6 +++---
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index b43361db..37871dd1 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -133,14 +133,14 @@
 
         <activity android:name=".selectcalendars.SelectVisibleCalendarsActivity"
             android:label="@string/select_visible_calendars_title"
-            android:theme="@android:style/Theme.Holo.Light"/>
+            android:theme="@android:style/Theme.Material.Light"/>
 
         <activity android:name=".selectcalendars.SelectSyncedCalendarsMultiAccountActivity"
             android:label="@string/select_synced_calendars_title"
-            android:theme="@android:style/Theme.Holo.Light"/>
+            android:theme="@android:style/Theme.Material.Light"/>
 
         <activity android:name="CalendarSettingsActivity" android:label="@string/preferences_title"
-            android:theme="@android:style/Theme.Holo.Light"/>
+            android:theme="@android:style/Theme.Material.Light"/>
 
         <!-- Declarations for search -->
         <!-- Make all activities a searchable context -->
@@ -163,10 +163,10 @@
 
         <!-- Declarations for alerts/reminders -->
         <activity android:name=".alerts.AlertActivity" android:launchMode="singleInstance"
-             android:theme="@android:style/Theme.Holo.Dialog" android:excludeFromRecents="true" />
+             android:theme="@android:style/Theme.Material.Dialog" android:excludeFromRecents="true" />
 
         <activity android:name=".alerts.QuickResponseActivity" android:launchMode="singleInstance"
-             android:theme="@android:style/Theme.Holo.Dialog" android:excludeFromRecents="true"
+             android:theme="@android:style/Theme.Material.Dialog" android:excludeFromRecents="true"
              android:label="@string/quick_response_dialog_title" />
 
         <receiver android:name=".alerts.AlertReceiver">
diff --git a/res/values/styles.xml b/res/values/styles.xml
index b24c3c3a..17d213a8 100644
--- a/res/values/styles.xml
+++ b/res/values/styles.xml
@@ -155,7 +155,7 @@
         <item name="android:paddingLeft">16dip</item>
     </style>
 
-    <style name="EditEvent_Layout" parent="android:Theme.Holo.Light">
+    <style name="EditEvent_Layout" parent="android:Theme.Material.Light">
     </style>
 
     <style name="TextAppearance.Date_Range_Title">
@@ -163,11 +163,11 @@
         <item name="android:textColor">@color/calendar_date_range_color</item>
     </style>
 
-    <style name="CalendarTheme" parent="android:Theme.Holo.Light">
+    <style name="CalendarTheme" parent="android:Theme.Material.Light">
         <item name="android:windowContentOverlay">@null</item>
     </style>
 
-    <style name="CalendarTheme.WithActionBar" parent="android:Theme.Holo.Light">
+    <style name="CalendarTheme.WithActionBar" parent="android:Theme.Material.Light">
         <item name="android:windowNoTitle">false</item>
         <item name="android:windowActionBar">true</item>
         <item name="android:windowBackground">@color/background_color</item>
-- 
2.17.1

